src/Microsoft.NET.Sdk.Functions.MSBuild/Targets/netstandard1.0/Microsoft.NET.Sdk.Functions.targets

```diff
@@ -12,7 +12,7 @@ WARNING:  DO NOT MODIFY this file unless you are knowledgeable about MSBuild and
          xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
 
   <PropertyGroup>
-    <_FunctionsTaskFramework Condition=" '$(MSBuildRuntimeType)' == 'Core'">netstandard2.0</_FunctionsTaskFramework>
+    <_FunctionsTaskFramework Condition=" '$(MSBuildRuntimeType)' == 'Core'">netcoreapp2.1</_FunctionsTaskFramework>
```

Comment:
why move from `netstandard` to `netcoreapp`? does that make a difference here?

Reply:
I didn't actually see this being used anywhere (is it?), so for consistency, I went ahead with the change. It's possible this could cause an issue if other tooling was depending on it, but with our defaults changing, this would no longer be correct if left alone.

Reply:
Nevermind, that's actually used to craft the path, so it would have to change if the path is being updated.